contributing.md

```diff
@@ -0,0 +1,50 @@
+## Contributing to terraform-terra
+
+Contributions to this repo can involve a number of moving parts, including
+
+* Config updates in other repos
```

Comment:
Maybe we can list the specific repos?
- terraform-shared
- dsp-ansible-configs
- firecloud-develop
- any new ansible roles or changes to existing ansible roles
Am I missing anything?